test/run_tests.js

```diff
@@ -80,6 +80,11 @@ const loadMocha = () => {
       // Skipping the file that manages shared state across tests.
       file => file !== path.join(__dirname, 'shared.js'),
     )
+    .filter(
+      file => file.indexOf(
```

Comment:
Please add a comment explaining what is being filtered, consistent with the other filters.

Reply:
Done.